libs/community/langchain_community/document_loaders/directory.py

```diff
@@ -129,6 +129,7 @@ def lazy_load(self) -> Iterator[Document]:
             path
             for path in paths
             if not (self.exclude and any(path.match(glob) for glob in self.exclude))
+            and path.is_file()
```

Comment:
Looks good. Ideally we'd remove the `is_file()` check later on since it involves a system call and is inefficient